lib/cc/services/flowdock.rb

```diff
@@ -13,11 +13,9 @@ class Config < CC::Service::Config
   self.description = "Send messages to a Flowdock inbox"
 
   def receive_test
-    notify("Test", repo_name, formatter.format_test)
-
-    { ok: true, message: "Test message sent" }
-  rescue => ex
-    { ok: false, message: ex.message }
+    notify("Test", repo_name, formatter.format_test).merge(
+      message: "Test message sent"
+    )
```

Comment:
Not really, no. The errors weren't being handled before, but they are now -- by the `WithErrorHandling` middleware. It's not in this method, but the previous version never did anything with the error anyway.